docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -0,0 +1,60 @@
+.. _lora_finetune_recipe_label:
+
```

Comment:
I'm not sure I 100% agree here - IMO these docs aren't necessarily aimed towards engineers who are used to quickly reading condensed information, but to maximise discovery of what we offer in torchtune - this was my primary motivation for writing these.

I will comb through the links and make sure they're relevant/necessary though : )

Reply:
Eh I do get @felipemello1's sentiment on this one. Esp because we immediately lead with 5 links. While many of them may be useful, I think we should instead lead with an example or something. Otherwise as a reader who just wants to understand how this recipe works/what it does I am immediately overwhelmed with pointers to literally half our live docs, making it hard to tease out what the actual relevant information is.

Reply:
I see now, sorry Felipe I don't think I fully grasped your original point : ) I'll address.

Reply:
I've updated to try remove the amount of noise and provide concrete examples, and leave additional information at the bottom.